plugin-server/src/cdp/services/hog-watcher.service.ts

```diff
@@ -22,6 +24,13 @@ export type HogWatcherFunctionState = {
     rating: number
 }
 
+export const hogFunctionExecutionTimeSummary = new Histogram({
+    name: 'cdp_hog_function_execution_duration_by_kind',
```

Comment:
We don't normally include the label name in the top name because you might add an additional label later and then the name wont make sense 
```suggestion
    name: 'cdp_hog_watcher_timings',
```

Also we have a similar metric elsewhere so maybe explicitly naming it todo with the watcher is better